server/routes/visit.routes.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import propertyVisitationSchema from '../schemas/visit.schema';
+import { schemaValidation, authenticate } from '../helpers/middleware';
+import VisitController from '../controllers/visit.controllers';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * /visit/book:
```

Comment:
To match the updated model name, schedule is preferable to be in sync with the design / UI context.
```suggestion
 * /visitation/schedule:
```